spring-data-neo4j/src/test/java/org/springframework/data/neo4j/examples/movies/repo/UserRepository.java

```diff
@@ -35,6 +35,7 @@
  * @author Michal Bachman
  * @author Luanne Misquitta
  * @author Gerrit Meier
+ * @author Michael J. Simons
  */
 @Repository
 public interface UserRepository extends PersonRepository<User, Long> {
```

Comment:
The file seems to use spaces instead of tabs. For consistency reasons I'd use them for the added lines, too, and rather switch to tabs at some point in time later.

Reply:
Eagle-Eye. I'll reformat the whole file, contains more tabs at the bottom.